### ISSUE TITLE
Console commands

### ISSUE THREAD

**Issue by yesitskev** [rgbkrk/pipboyrelay#11]
More of a question than an issue,

Is there any currently documented discoveries of issueing console commands through the pipboy API?

**Comment by rgbkrk** [comment]
Yup, the library you want to use is [pipboylib](https://github.com/RobCoIndustries/pipboylib) which has some examples and relevant links to the pipboy spec.
